### ISSUE TITLE
SimpleCookie code fails in python 2.6.4

### ISSUE THREAD

**Issue by lost-theory** [jzempel/flask-exceptional#3]
Flask-Exceptional dies when setting cookie values in 2.6.4:

``` python
Traceback (most recent call last):
  File "/var/home/steven-kryskalla/project/lib/python2.6/site-packages/Flask-0.7dev_20110406-py2.6.egg/flask/app.py", line 1014, in __call__
    return self.wsgi_app(environ, start_response)
  File "/var/home/steven-kryskalla/project/lib/python2.6/site-packages/Flask-0.7dev_20110406-py2.6.egg/flask/app.py", line 1002, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/var/home/steven-kryskalla/project/lib/python2.6/site-packages/flaskext/exceptional.py", line 144, in ret_val
    self._post_data(context)
  File "/var/home/steven-kryskalla/project/lib/python2.6/site-packages/flaskext/exceptional.py", line 180, in _post_data
    "request": self.__get_request_data(context.app, context.request, context.session),
  File "/var/home/steven-kryskalla/project/lib/python2.6/site-packages/flaskext/exceptional.py", line 294, in __get_request_data
    headers["Cookie"] = cookie.output(header='', sep=';').strip()
  File "/var/home/steven-kryskalla/Python-2.6.4/Lib/Cookie.py", line 594, in output
    result.append( V.output(attrs, header) )
AttributeError: 'unicode' object has no attribute 'output'
```

This happens because Cookie.py in the 2.6.4 stdlib has a bug that is fixed in later versions:

http://bugs.python.org/issue5275
http://hg.python.org/cpython/rev/c51721d64451/

Maybe that `cookie[k] = v` code can be used as a workaround for older versions of python.

**Comment by jzempel** [comment]
Thanks for raising the issue.  I'll probably take a closer look at it early
next week.

Cheers,
Jonathan.

**Comment by jzempel** [comment]
Closed with pull request #4.
